.changeset/curly-moons-sin.md

```diff
@@ -0,0 +1,5 @@
+---
+'create-svelte': minor
```

Comment:
```suggestion
'create-svelte': major
```

For something like swapping out one plugin for another, I think we should err on the side of caution here. We don't know whose programmatic use of create-svelte this is going to impact. And the major version bump won't affect anyone using it as a CLI anyway.

Reply:
OK!